### DISCUSSION TITLE
Some description are missing

### DISCUSSION THREAD

**Discussion by danieleavolio** [deliton/eldenring-api#14] [General]
I was trying to build a simple items search web app, but while getting some items' description, I noticed that some descriptions were half the real length or even null (see Hoarah Loux's grace **null** and Elden Beast's one was like half the length

**Comment by deliton** [comment]
Yeah, we still have a lot of missing data here and there. If you want to contribute with data, ping me so I can assist you with the process.

**Reply by danieleavolio** [reply]
Since I am not an expert on doing something collective on GitHub, how can I help you?
Telling you which items are missing or something like that?

**Reply by deliton** [reply]
All the data available in this API is public and can be modified. If you want to add missing items or data, you can go to `api/public/data` and there you'll find the JSON files with all the data from this API. Then, you can fork this repository, add the data you want in your forked repository and then you can open a Pull Request for me to analyze. I'll create a CONTRIBUTING.md file with more instructions and a video soon
